dds_web/security/project_user_keys.py

```diff
@@ -41,43 +71,47 @@ def __decrypt_with_rsa(ciphertext, private_key):
 
 
 def __encrypt_project_private_key(owner, project_private_key):
-    public_key = serialization.load_der_public_key(owner.public_key)
-    if isinstance(public_key, asymmetric.rsa.RSAPublicKey):
-        return __encrypt_with_rsa(project_private_key, public_key)
-    # TODO: Change exception type
-    exception = Exception("Public key cannot be loaded for encrypting the project private key!")
-    flask.current_app.logger.exception(exception)
-    raise exception
-
-
-def __decrypt_project_private_key(user, encrypted_project_private_key):
-    user_private_key = serialization.load_der_private_key(
-        __decrypt_user_private_key(user), password=None
-    )
-    if isinstance(user_private_key, asymmetric.rsa.RSAPrivateKey):
-        return __decrypt_with_rsa(encrypted_project_private_key, user_private_key)
-    exception = Exception("User private key cannot be loaded!")
-    flask.current_app.logger.exception(exception)
-    raise exception
+    if owner.public_key is None:
```

Comment:
Personally I like `if not owner.public_key` better but this is a tiiiny little thing and I'm fine with keeping it as it is now

Reply:
I don't have a strong opinion, but as I have learned from the Internet, `not owner.public_key` will check a lot different things in addition to None check. `is None` check seemed straightforward to me. If you want I can change it?

Reply:
changed to your preference